pkg/controllers/tlspolicy/tlspolicy_controller.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
+	v1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
```suggestion
	certmanv1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
```

Avoids confusion with our own v1 api, and consistent with other places we import it https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/pkg/controllers/tlspolicy/tlspolicy_certmanager_certificates.go#L8

Reply:
Does make me wonder if this is the right place for this validation :thinking:

Reply:
I moved it to a certmanager_helper file.. lmk if you think that's suitable.